src/theme.tsx

```diff
@@ -6,7 +6,7 @@ import type {Skin, Colors, SkinName, TextPresetsConfig} from './skins/types';
 import type {TrackingEvent} from './utils/types';
 import type {MediaQueries} from './utils/media-queries';
 
-export type ThemeTexts = typeof TEXTS_ES;
+export type ThemeTexts = Readonly<typeof TEXTS_ES>;
```

Comment:
Please verify the result en webapp:
![image](https://user-images.githubusercontent.com/579417/201067849-d016e836-bf71-4052-acf6-41de3d1408a2.png)

What I do is:
1. build mistica: `yarn build`
2. `cp -r dist dist-es ../webapp/node_modules/@telefonica/mistica/`
3. in webapp: run `yarn flow` (sometimes I must `killall flow` first)

Reply:
Sure, done here: https://github.com/Telefonica/webapp/pull/2303

Reply:
Done on the CMS, with no errors from this PR (I've detected this: https://github.com/Telefonica/mistica-web/pull/528#discussion_r1018979437)